packages/components/modal/src/index.ts

```diff
@@ -42,3 +42,4 @@ export { ModalOverlay } from "./modal-overlay"
 export type { ModalOverlayProps } from "./modal-overlay"
 export { useModal } from "./use-modal"
 export type { UseModalProps, UseModalReturn } from "./use-modal"
+export { modalManager, useModalManager } from "./modal-manager"
```

Comment:
I would export only the hook in this case.

```suggestion
export { useModalManager } from "./modal-manager"
```